guide/creating-your-bot/README.md

```diff
@@ -121,8 +125,21 @@ You can specify quite intricate patterns in `.gitignore` files, check out the [G
 
 Open your code editor and create a new file. We suggest that you save the file as `index.js`, but you may name it whatever you wish.
 
-Here's the base code to get you started:
+```:no-line-numbers {4}
+discord-bot/
+├── node_modules
+├── config.json
+├── index.js // <-- Create a new file named index.js
+├── package-lock.json
+└── package.json
+```
+
+(We will be using `config.json` to store our token for the rest of the guide)
```

Comment:
a lot of people who come to the djs server asking for help end up mixing their `index.js` and `deploy-commands.js` files. most likely because they didnt read carefully and just skimmed through.

i figured we could avoid this mistake by simply adding a code block that shows the files in the project directory. This same code block is shown again in `creating-commands.md` on line 34 when we tell the reader to create a new file called `deploy-commands.js`.